src/main/java/core/basesyntax/service/ColorSupplier.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.service;
+
+public class ColorSupplier {
```

Comment:
The `ColorSupplier` class should include a method to return a random color from the `Color` enum. Consider implementing a method like `public Color getRandomColor()` that randomly selects and returns a color.